extension/src/NABfunctions.ts

```diff
@@ -1121,3 +1123,123 @@ async function handleInvalidXmlError(error: unknown): Promise<void> {
     error.length
   );
 }
+
+export function getHoverText(
+  document: vscode.TextDocument,
+  position: vscode.Position
+): vscode.MarkdownString[] {
+  if (!SettingsLoader.getSettings().enableTranslationsOnHover) {
```

Comment:
Declare `const settings` since it's being reused on line 1155

Reply:
done